pyproject.toml

```
@@ -0,0 +1,74 @@
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+# Check https://flit.readthedocs.io/en/latest/pyproject_toml.html for all available sections
+name = "pyansys"
+version = "2023.1.dev0"
+description = "Pythonic interfaces to Ansys products"
+readme = "README.rst"
+license = "MIT"
+homepage = "https://github.com/pyansys"
+repository = "https://github.com/pyansys/pyansys_landing"
+documentation = "https://docs.pyansys.com/"
+authors = [
+    "ANSYS, Inc. <pyansys.support@ansys.com>",
+]
+maintainers = [
+    "PyAnsys developers <pyansys.maintainers@ansys.com>",
+]
+classifiers = [
+    'Development Status :: 4 - Beta',
+    'Intended Audience :: Science/Research',
+    'Topic :: Scientific/Engineering :: Information Analysis',
+    'License :: OSI Approved :: MIT License',
+    'Operating System :: Microsoft :: Windows',
+    'Operating System :: POSIX',
+    'Operating System :: MacOS',
+    'Programming Language :: Python :: 3.7',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+]
+
+packages = [
+    { include = "pyansys", from = "src" },
+]
+
+[tool.poetry.dependencies]
+python = ">=3.7,<4.0"
+importlib-metadata = {version = "^4.0", python = "<3.8"}
+ansys-mapdl-core = "==0.63.0"
+ansys-dpf-core = "==0.4.2"
+ansys-dpf-post = "==0.2.2"
+ansys-fluent-core = "==0.10.1"
+pyaedt = "==0.5.11"
+ansys-platform-instancemanagement = "==1.0.2"
+ansys-grantami-bomanalytics = "==1.0.1"
+ansys-openapi-common = "==1.1.1"
+
+# Optional dependencies
+ansys-mapdl-reader = {version = "==0.51.14", optional = true}

```

With https://github.com/pyansys/pymapdl-reader/pull/147 and https://github.com/pyansys/pymapdl-reader/actions/runs/2794314543, this should work now.

```suggestion
ansys-mapdl-reader = {version = "==0.51.15", optional = true}
```

---

Thanks @akaszynski for your work!